src/main/java/com/google/maps/GeoApiContext.java

```diff
@@ -54,6 +56,7 @@
    */
   public interface RequestHandler {
     <T, R extends ApiResponse<T>> PendingResult<T> handle(String hostName, String url, String userAgent, Class<R> clazz, FieldNamingPolicy fieldNamingPolicy, long errorTimeout);
+    <T, R extends ApiResponse<T>> PendingResult<T> handlePost(String hostName, String url, String jsonPayload, String userAgent, Class<R> clazz, FieldNamingPolicy fieldNamingPolicy, long errorTimeout);
```

Comment:
Technically a POST payload, particularly of type String, could be non-JSON. Can you rename the jsonPayload param to something more generic, e.g. payload?